src/viam/app/app_client.py

```diff
@@ -375,6 +375,41 @@ class AppClient:
 
     Constructor is used by `ViamClient` to instantiate relevant service stub. Calls to `AppClient`  methods should be made through
     `ViamClient`.
+
+    Establish a Connection::
+
+        import asyncio
+
+        from viam.rpc.dial import DialOptions, Credentials
+        from viam.app.viam_client import ViamClient
+
+
+        async def connect() -> ViamClient:
+            dial_options = DialOptions(
+            credentials=Credentials(
+                type="api-key",
+                # Replace "<API-KEY>" (including brackets) with your API key
+                payload='<API-KEY>',
+            ),
+            # Replace "<API-KEY-ID>" (including brackets) with your API key
+            # ID
+            auth_entity='<API-KEY-ID>'
+            )
```

Comment:
We have a helper function for this to simplify this a bit, 
`dial_options = DialOptions.with_api_key(<API-KEY>, <API-KEY-ID>)`